Add a continuation for James after the intro missions (#5089)
endless-sky/endless-sky: data/human/transport missions.txt

			branch medicineless
				not "james medicine: medicine taken"
			`	In the height of the moment, you remember that when you were buying medicine for James, you bought some for yourself as well. You reach for the box and open it. Inside are a dozen or so neatly-arranged amber-colored pills and a long list of instructions shouting "EXPERIMENTAL" and "UNTESTED" in bold red letters.`
			choice
				`	(Take a pill.)`
				`	(Think of other solutions.)`
					goto medicineless
			`	You hastily take one of the pills and swallow it, as per the instructions on the box. You can already start feeling better, even though the dizziness does not go away immediately. Just in case, you switch the autopilot on. The last thing you see is the ship's display reading "Landing on <origin>" as you briefly close your eyes.`
			`	When you wake up, spaceport workers on <origin> have just started refueling your ship.`
				accept
			label medicineless
			branch crewless

> 				"flagship crew" < 1

Reviewer: You might want to consider changing the number here to more than one, since you imply later during the crew conversation that there are multiple crew members besides youself- enough, in fact, that "a few" is a portion of the crew and not the entire crew itself.

			`The Testament enters the planet's atmosphere at high speed and lands somewhat uncontrolled near a waste processing plant. The cargo hold is glowing red hot and some areas in the cargo hold appear to be on fire.`
			`	Veronica disembarks, walks over to you, and apologizes for the difficult trip. She explains that the cooling systems failed, causing the sushi calamity you experienced during the multiple legs of your journey.`
			`	You are paid <payment> and she asks you not to talk about this escort job to anybody. You leave the docks quickly in order to escape the horrible smell of overheated fish that is coming from Veronica and her ship.`

mission "James Medicine [0]"
	minor
	name "Medicine from <planet>"

> 	description "James's doctors have asked you to bring some newly-developed remedy from <planet> to <origin> as quickly as possible, even though they are not really sure what his illness is."

Reviewer 2: ```suggestion
	description "James' doctors have asked you to bring some newly-developed remedy from <planet> to <origin> as quickly as possible, even though they are not really sure what his illness is."
```
doctors is plural, so this should be `James'`. [Ref](https://data.grammarbook.com/blog/apostrophes/apostrophes-with-names-ending-in-s-ch-or-z/)

Author: James is not plural for Jame. The number of a possessive depends on the owner, not on the owned thing.
Consider:
* The performer's instrument (a piano and a piano player)
* The performer's instruments (a piano, a violin, and a talented artist)
* The performers' instruments (a piano, a piano player, a violin, and a violin player)
* The performers' instrument (a piano and two piano players using it at once).

Reviewer 3: Eh, no. It's James' doctor/s.
As long as you have an s before the single quote, no s is required at the back.

Author: To quote [grammarly](https://www.grammarly.com/blog/possessive-case/):
> Do we add another s for the possessive form of a name ending with s? Which is correct, Chris’s chair or Chris’ chair? James’s car or James’ car? Actually, both ways are correct. If a proper name ends with an s, you can add just the apostrophe or an apostrophe and an s.

Now, in the Endless Sky, there are cases of both. Consider:
* `Davy Jones' Locker`, from `human/names.txt` / `human/hails.txt`
* `Lieutenant Paris' voice` from `human/deep missions.txt`
* `Paris's voice` from `human/deep missions.txt`
* `Ijs's office` from `human/free worlds reconciliation.txt`
* `James's face` from `intro missions.txt`.

In general, my personal opinion is that `James's` is better than `James'`, but if someone pushes hard enough for it, I might change it :smiley:.

> 	description "James's doctors have asked you to bring some newly-developed remedy from <planet> to <origin> as quickly as possible, even though they are not really sure what his illness is."

Reviewer 4: ```suggestion
	description "James' doctors have asked you to bring some newly-developed remedy from <planet> to <origin> as quickly as possible, even though they are not really sure what his illness is."
```

	deadline
	source "Hestia"
	destination "Nimbus"
	to offer
		has "main plot completed"
		or
			has "Intro [3 Transport]: done"
			has "Intro [3 Freighter]: done"
			has "Intro [3 Interceptor]: done"
	on offer
		conversation

> 			`Finding yourself on <origin>, you smile at the memory of having dropped James here after he gave you some basic spaceflight instructions. Do you wish to visit him and see how he has been faring?`

Reviewer 2: ```suggestion
			`Finding yourself on <origin>, you smile at the memory of having dropped off James here after he gave you some basic spaceflight instructions. Do you wish to visit him and see how he has been faring?`
```
Just a suggestion

			choice
				`	(Yes.)`
				`	(Not right now.)`
					defer
			`	Unlike some other planets, where it is practically impossible to find a person without having contact information, <origin> has a planetary database. You eagerly walk over to the nearest information desk and wait for the operator to make a few calls. At last, you are directed to a small, respected establishment, the Pilot Veteran Clinic. You can only wonder what James is doing there.`

> 			`	The clinic is housed in a small modern-looking building, which looks cunningly similar to a giant Firebird. Inside, the rooms are arranged to resemble bunk rooms on a spaceship, likely so that the patients would feel right at home.`

Reviewer 5: ```suggestion
			`	The clinic is housed in a small, modern-looking building, which looks cunningly similar to a giant Firebird. Inside, the rooms are arranged to resemble bunk rooms on a spaceship, likely so that the patients would feel right at home.`
```

> 			`	The clinic is housed in a small, modern-looking building, which looks cunningly similar to a giant Firebird. Inside, the rooms are arranged to resemble bunk rooms on a spaceship, likely so that the patients would feel right at home.`

Reviewer 2: Not sure if `cunningly` is fitting here.

Author: As per https://www.thefreedictionary.com/cunning:
> 2. Executed with or exhibiting ingenuity.

The clinic is not just an ordinary square/rectangular building. It was built from the ground in order to be a clinic for veteran pilots. What better way to show that off, apart from making it shaped like a iconic ship which every veteran pilot would have flown?

I'd be willing to accept a better adverb there. But I still think shows some ingenuity to architecture a building after its purpose.

Reviewer: "Cunningly" has a negative connotation to it, which makes it off-putting here. Maybe the second half of the phrase could be rewritten to "designed to look like a giant Firebird"? That conveys the intent to make it look like a Firebird while removing the "cunningly".

> 			`	As it is currently lunch time, there is no one in sight at the clinic. Luckily, the operator gave you the room number, which you quickly reach after a few minutes.`

Reviewer 2: `quickly after a few minutes`?

Author: Marking as resolved to keep the amount of unresolved comments low; will fix with the comment below.

> 			`	There is no one in sight at the clinic, probably because it is around lunch time. Luckily, the operator gave you the number of James's room, and you manage to find the correct door after a few minutes.`

Reviewer 4: ```suggestion
			`	There is no one in sight at the clinic, probably because it is around lunch time. Luckily, the operator gave you the number of James' room, and you manage to find the correct door after a few minutes.`
```

			choice

> 				`	(Wait for someone to show up.)`

Reviewer 2: Since you're apparently standing at the door, it would kinda make sense to reach the door rather than the room. reading the last sentence, i thought i had already entered.

Author: I'm a bit unsure about what the issue is here. Should I change the "which" to a "the door of which" or something like that?

Reviewer 2: The issue is that i read the sentence and was like "yeah now i'm in the room" - only to realize i'm actually outside the door.

I'd probably rephrase it to something like "After a few minutes [see comment above] you find the correct door."

					goto wait
				`	(Enter the room.)`
			apply
				set "james medicine: infected"

> 			`	You enter the room and find that James is inside, lightly asleep. He wakes up just as you close the door.`

Reviewer 4: Seems a bit odd to have the room of a diseased person unlocked.

Author: Fix the later text a bit; for clarification, what I'm thinking is that the policy demands patients with diseases known to be contagious to be kept isolated, but does not mandate similar measures for patients with unknown diseases. So, while you are not expected or generally allowed in the room, you can enter it without running into locked doors.

You (the player) just manage to pass by the front door without meeting anyone at a reception or similar. Given that you don't get to speak to anyone else but one lone doctor, might hint that the establishment, while respected, is understaffed, and the staff it has is too easy-going to bother with security.

> 			`	"<first>! Fancy meeting you here! I heard you became a war hero of sorts?", he adds with a wink.`

Reviewer 4: ```suggestion
			`	"<first>! Fancy meeting you here! I heard you became a war hero of sorts?" he adds with a wink.`
```

			choice
				`	"Nothing special, I just did what is right."`
				`	"Quite so! Seems like I saved the galaxy this time."`

> 			`	"Yeah, yeah... You know, I am feeling pretty weak now, and while the doctors say that I will live through this illness, there is still a chance that it would be my last."`

Reviewer 4: The jump from talking about your triumphs to James' illness feels pretty abrupt.

> 			`	"So, I wanted to tell you something..", he rambles on, " just to make sure that knowledge is not lost. While I was exploring the planets and systems reachable from human space, I stumbled upon Ultima Thule, due northeast of here. At the time I piloted just a small warship, so I couldn't brave myself to pass through it, but.. there is a wormhole there!"`

Reviewer 5: ```suggestion
			`	"So, I wanted to tell you something..", he rambles on, " just to make sure that knowledge is not lost. While I was exploring the planets and systems reachable from human space, I stumbled upon Ultima Thule, due northeast of here. At the time I piloted just a small warship, so I couldn't muster the courage to pass through it, but.. there is a wormhole there!"`
```

> 			`	"So, I wanted to tell you something..", he rambles on, " just to make sure that knowledge is not lost. While I was exploring the planets and systems reachable from human space, I stumbled upon Ultima Thule, due northeast of here. At the time I piloted just a small warship, so I couldn't muster the courage to pass through it, but.. there is a wormhole there!"`

Reviewer 4: ```suggestion
			`	"So, I wanted to tell you something..." he rambles on, "just to make sure that knowledge is not lost. While I was exploring the planets and systems reachable from human space, I stumbled upon Ultima Thule, due northeast of here. At the time I piloted just a small warship, so I couldn't muster the courage to pass through it, but... there is a wormhole there!"`
```

			branch interesting
				not "First Contact: Hai: offered"
			choice
				`	(Tell him about the Hai.)`
				`	(Say nothing about it.)`
					goto interesting
			`	"I have actually been there," you say, "It leads to a nearby part of space inhabited by the Hai, an advanced alien civilization of squirrel-like creatures."`

> 			`	"Indeed? Are they friendly?"`

Reviewer 2: The second time `Indeed` has been used in a couple sentences (the other being your reply to the war hero remark). Given that these 2 aren't really coming from academic families, i doubt they'd use this word at all - and if they do, probably just once.

Author: Removed all instances of "Indeed". Indeed, it seems I used that word too often.

> 			`	"I have actually been there," you say, "It leads to a nearby part of space inhabited by the Hai, an advanced alien civilization of squirrel-like creatures."`

Reviewer 4: ```suggestion
			`	"I have actually been there," you say, "It leads to a nearby part of space inhabited by the Hai, an advanced alien civilization of squirrel-like creatures."`
```

Author: Uhh, should something be changed here?

Author: I'm not sure about putting a period there; going to check other conversations and grammar sites...

			choice

> 				`	"Yes, they even offer humans to settle there or buy their technology."`

Reviewer 5: ```suggestion
				`	"Yes, they even offer humans permission to settle there or buy their technology."`
```

> 				`	"Yes, they even offer humans permission to settle there or buy their technology."`

Reviewer 2: ```suggestion
				`	"Yes, they even grant humans permission to settle there or buy their technology."`
```
or
```suggestion
				`	"Yes, they even allow humans to settle there or buy their technology."`
```

					goto doctor
				`	"Yes, though I'm worried they can conquer all of human space if they wish."`
					goto doctor
			label interesting

> 			`	"Sounds interesting," you say, "Could it lead to some aliens, or perhaps to some worlds waiting to be settled?"`

Reviewer 2: might want to use
```suggestion
			`	"Sounds interesting," you say, "Could it lead to some aliens, or perhaps to some worlds waiting to be colonized?"`
```
settle is more of an individual action.

Author: Seems like I couldn't think of that word in time when writing it. Thanks :)

> 			`	"Sounds interesting," you say, "Could it lead to some aliens, or perhaps to some worlds waiting to be colonized?"`

Reviewer 4: ```suggestion
			`	"Sounds interesting," you say. "Could it lead to some aliens, or perhaps to some worlds waiting to be colonized?"`
```

> 			`	"Not sure. It is located in a lawless barren of space, however, so I wouldn't be surprised if there pirates or even Alphas on the other side."`

Reviewer 5: ```suggestion
			`	"Not sure. It is located in a lawless barren region of space, however, so I wouldn't be surprised if there pirates or even Alphas on the other side."`
```

Author: Not really necessary (barren can be a noun), but still sounds good.

> 			`	"Not sure. It is located in a lawless barren region of space, however, so I wouldn't be surprised if there pirates or even Alphas on the other side."`

Reviewer 2: ```suggestion
			`	"Not sure. It is located in a lawless barren region of space, however, so I wouldn't be surprised if there are pirates or even Alphas on the other side."`
```

			label doctor
			`	Just as the conversation hangs for a moment, the door opens and a doctor in protective clothing steps in.`

> 			`	"Who are you? And why are you in this room?", the doctor inquires, in a soft but authoritative voice.`

Reviewer 4: ```suggestion
			`	"Who are you? And why are you in this room?" the doctor inquires, in a soft but authoritative voice.`
```

			choice
				`	"I am <first> <last>, here to see this friend of mine."`

> 				`	"I was.. just strolling around."`

Reviewer 4: ```suggestion
				`	"I was... just strolling around."`
```

> 			`	"Nevermind what you were doing. I have to warn you that the patient here has a disease we do not know much about yet. It could potentially be some deadly, contagious disease. Now, please make haste and remove yourself from the room, so as to not risk catching it yourself."`

Reviewer 2: Wow, doc doesn't sugar coat! :D

> 			`	"Nevermind what you were doing. I have to warn you that the patient here has a disease we do not know much about yet. It could potentially be some deadly, contagious disease. Now, please make haste and remove yourself from the room, so as to not risk catching it yourself."`

Reviewer 4: Maybe don't repeat "disease" twice here.

> 			`	Following the doctor's instructions, you quickly exit the room. As you close the door, you can make something of a chuckle, probably from James.`

Reviewer 5: ```suggestion
			`	Following the doctor's instructions, you quickly exit the room. As you close the door, you can make out something of a chuckle, probably from James.`
```
```suggestion
			`	Following the doctor's instructions, you quickly exit the room. As you close the door, you hear something of a chuckle, probably from James.`
```

> 			`	Following the doctor's instructions, you quickly exit the room. As you close the door, you hear something of a chuckle, probably from James.`

Reviewer 2: If we're dealing with this kinda doctor, why not let him shove us out the room while we still try to process the news?

				goto end
			label wait
			`	Soon enough, a doctor passes by, heading for the room.`

> 			`	"Good day! How can I help you?", he inquires.`

Reviewer 4: ```suggestion
			`	"Good day! How can I help you?" he inquires.`
```

			`	"I just wanted to check on a friend of mine, James, who I heard is in this room. May I see him?"`

> 			`	"Well, he is infected by some disease. We are not sure what it is exactly and if it is contagious or not. So I would recommend not seeing him in person, at least for the moment," he says just before entering James's room.`

Reviewer 2: ```suggestion
			`	"Well, he is infected with some disease. We are not sure what it is exactly and if it is contagious or not. So I would recommend not seeing him in person, at least for the moment," he says just before entering James's room.`
```

> 			`	"Well, he is infected with some disease. We are not sure what it is exactly and if it is contagious or not. So I would recommend not seeing him in person, at least for the moment," he says just before entering James's room.`

Reviewer 4: ```suggestion
			`	"Well, he is infected with some disease. We are not sure what it is exactly and if it is contagious or not. So I would recommend not seeing him in person, at least for the moment," he says just before entering James' room.`
```

			label end
			choice
				`	(Wait outside the room.)`
			`	Around half an hour later, the doctor finally exists the room.`
			`	"I see you probably want to help," he says, "I need someone to quickly go to the spaceport and find a captain willing to bring some newly-discovered medication from <destination> as quickly as possible. Can you do that?"`
			choice
				`	"Actually I am a spaceship captain. I could do that."`
				`	"Yes, I can find a capable spaceship captain."`
					goto `second chance`

> 				`	"What am I, a message-boy? Find someone yourself."`

Reviewer 2: And \<first> \<last> doesn't take any shit.

Author: I might change this to be a bit less rude.

> 				`	"I'm to busy to run errands. Ask someone else."`

Reviewer 4: ```suggestion
				`	"I'm too busy to run errands. Ask someone else."`
```

					decline

> 			`	"Indeed? Superb! Come back as soon as possible. We wouldn't want to risk your friend's life for a minute delay."`

Reviewer 2: Third indeed - i still don't think it fits here, but a doctor would be the person to use it.

				accept
			label `second chance`

> 			`	On the way to the spaceport you realize that you do not have to find another spaceship captain to bring medicine from <planet>. You could do it yourself!`

Reviewer 4: Why even give the option to start looking then if the player is going to just "realize" they can do it on their own?

			choice
				`	(Take the job.)`
					accept
				`	(Find someone else.)`
					decline
	on accept
		event "james medicine too slow" 20
	npc
		government "Pirate"
		personality nemesis staying harvests plunders
			confusion 20
		system "Alpha Hydri"
		fleet "Small Northern Pirates"

event "james medicine too slow"

mission "James Medicine [1]"
	landing
	name "Medicine from <origin>"

> 	description "James's doctors have asked you to bring some newly-developed remedy from <planet> to <origin> as quickly as possible, even though they are not really sure what his illness is."

Reviewer 4: The origin and planet of the previous mission are now swapped, so this description needs to swap the text replacements.
```suggestion
	description "James' doctors have asked you to bring some newly-developed remedy from <origin> to <planet> as quickly as possible, even though they are not really sure what his illness is."
```

	cargo "Essential medicine" 1
	deadline
	source "Nimbus"
	destination "Hestia"
	to offer
		has "James Medicine [0]: done"
	blocked "You have reached <origin>, but you need <capacity> in order to take the medicine for James. Return here when you have the required space free."
	on visit
		dialog "generic cargo on visit"
	on offer

> 		dialog `You take the medicine from a newly-constructed section of the spaceport, as instructed. Now you just have to deliver it back to <planet>.`

Reviewer 4: This should probably be a conversation as dialogs always give the option to accept/decline if in an on offer block.

Author: I wonder, would it work if it is a dialog in `on complete` of the previous mission, while this one is automatically accepted?

	on complete
		conversation
			`Landing again on <planet>, you hurry to the Pilot Veteran Clinic with the medicine from <origin>.`
			branch slow

> 				has "event: james medicine too slow"

Reviewer 2: what's the point of this event?

Author: It makes it so that if you complete the mission with a jump drive, you get the "better" ending, but if you complete it only based on the deadline (or otherwise with just a hyperdrive), you get the "bad" ending.

> 			`	The doctor that spoke to you earlier appears soon after you enter the building. "Back already? I can hardly believe the news!" he exclaims and adds with a smile, "I have no idea how you made the trip so fast, but this little box you brought might be life-saving for your friend."`

Reviewer 4: ```suggestion
			`	The doctor that spoke to you earlier appears soon after you enter the building. "Back already? I can hardly believe the news!" he exclaims and adds with a smile. "I have no idea how you made the trip so fast, but this little box you brought might be life-saving for your friend."`
```

Author: I'm pretty sure this suggestion is incorrect. Maybe if `and adds with a smile` is dropped as well, a dot there could be good.

			choice
				`	"I did my best."`
					goto end
				`	"No help is too much for a friend."`
					goto end
			label slow

> 			`	The doctor that spoke to you earlier appears soon after you enter the building. "Alas! Such is our fate, to slowly pass from this life to the next!", he exclaims, and adds "Unfortunately, the disease was more onerous than we could have anticipated. The medicine you brought would save James's life, but the damage already wrought would leave him unable to walk freely."`

Reviewer 2: Kinda weird how he knows that know, when just prior it was completely alien to them.

Author: There is a potential rationale that they were in communication with the doctors in Nimbus and are now certain that the medicine would work.

Otherwise, I might just change the "would" into a "might"..

> 			`	The doctor that spoke to you earlier appears soon after you enter the building. "Alas! Such is our fate, to slowly pass from this life to the next!", he exclaims, and adds "Unfortunately, the disease was more onerous than we could have anticipated. The medicine you brought might save James's life, but the damage already wrought would leave him paralysed forever."`

Reviewer 4: ```suggestion
			`	The doctor that spoke to you earlier appears soon after you enter the building. "Alas! Such is our fate, to slowly pass from this life to the next!" he exclaims, and adds, "Unfortunately, the disease was more onerous than we could have anticipated. The medicine you brought might save James's life, but the damage already wrought would leave him paralysed forever."`
```

			apply
				set "james medicine: james disabled"
			choice
				`	"I tried my best."`
					goto end

> 				`	"It's your fault, not delivering for the medicine earlier."`

Reviewer 5: ```suggestion
				`	"It's your fault, not sending for the medicine earlier."`
```
```suggestion
				`	"It's your fault, not ordering the medicine earlier."`
```
```suggestion
				`	"It's your fault, not requesting the medicine earlier."`
```

> 			`	The doctor seems amazed by your complaint, even though he does not reply to it in any way.`

Reviewer 2: suggestion
```suggestion
			`	The doctor seems [stunned, staggered] by your complaint, but does not reply to it in any way.`
```

Author: I cannot commit the suggestion directly that way (unless the doctor is a Wanderer). Will fix it when I get to edit things outside of GH's web interface.

Author: Changed it somewhat, making the doctor a bit more human. Hopefully it is better that way.

			label end
			branch noname
				not "james medicine: infected"

> 			`	"Anyway, we have to pay for the services we receive. Here, take <payment>", he says and drops his voice to mention, "I would not recommend seeing James now, though I will surely tell him how helpful you have been, <last>."`

Reviewer 5: ```suggestion
			`	"Anyway, we have to pay for the services we receive. Here, take <payment>", he says and drops his voice to mention, "I would not recommend seeing James now, though I will surely tell him how helpful you have been, Captain <last>."`
```
Just referring to someone by their last name can come across as slightly rude.

Author: I'm not sure if Captain <last> is capitalized in other missions.

> 			`	"Anyway, we have to pay for the services we receive. Here, take <payment>", he says and drops his voice to mention, "I would not recommend seeing James now, though I will surely tell him how helpful you have been, Captain <last>."`

Reviewer 4: ```suggestion
			`	"Anyway, we have to pay for the services we receive. Here, take <payment>," he says and drops his voice to mention, "I would not recommend seeing James now, though I will surely tell him how helpful you have been, Captain <last>."`
```

				goto finish
			label noname

> 			`	"Anyway, we have to pay for the services we receive. Here, take <payment>", he says and drops his voice to mention, "I would not recommend seeing James now, but maybe after he is well again."`

Reviewer 4: ```suggestion
			`	"Anyway, we have to pay for the services we receive. Here, take <payment>," he says and drops his voice to mention, "I would not recommend seeing James now, but maybe after he is well again."`
```

			choice

> 				`	"Sure! Can you him that captain <last> was looking for him?"`

Reviewer 4: ```suggestion
				`	"Sure! Can you tell him that Captain <last> was looking for him?"`
```

				`	"Oh yeah? I am going to visit him regardless."`
					goto visit
			`	"Of course, at the first possible occasion."`
				goto finish
			label visit

> 			`	The doctor doesn't look very happy, but lets you go as there is nothing in the rules which would prevent you visiting James.`

Reviewer 4: Why? Dude's got a potentially contagious disease. I'd think there'd be a rule against meeting him until more is figured out.

			apply
				set "james medicine: infected"
			`	You enter James's room but find him tight asleep. Not wanting to wake him up, you leave a note urging him to contact you when he is well again.`

> 			`	Just before you leave the room, you notice a note written by him on one of the chairs. It reads "The North; Ultima Thule", leaving the reader to decide what its meaning is.`

Reviewer 2: I kinda lost track - does this also show if we talked about Ultima Thule earlier?

Author: `james medicine: infected` is set only if you enter the room, so, no.

> 			`	Just before you leave the room, you notice a note written by him on one of the chairs. It reads "The North; Ultima Thule", leaving the reader to decide what its meaning is.`

Reviewer 4: ```suggestion
			`	Just before you leave the room, you notice a note written by him on one of the chairs. It reads "The North; Ultima Thule," leaving the reader to decide what its meaning is.`
```

> 			label finish

Reviewer 4: Instead of having a dead label at the end of a conversation, you can use any conversation endpoint (e.g. accept, decline) in an on complete conversation and it will end the conversation without consequence (as endpoints only have effects in on offer conversation).

Author: Probably this was a leftover from a previous version of the text; going to to remove it :smiley:

> 		payment 150000

Reviewer 4: Put the payment above the conversation.

	
	npc
		government "Pirate"
		personality nemesis staying harvests plunders
			confusion 20
		system "Alpha Hydri"
		fleet "Small Northern Pirates"

mission "James Medicine [1] Preventive"
	# Name is important; it should be lexically ordered after "James Medicine [1]"
	invisible
	landing
	source "Nimbus"
	to offer
		has "James Medicine [0]: done"
		has "james medicine: infected"
	on offer
		dialog
			`Thinking about what the doctor said, it might be wise to buy some of the medicine for yourself. Checking the receipt you got, this would cost you 10,000 credits.`
	on accept

> 		payment -10000

Reviewer 4: If the player doesn't have enough credits, they'll be able to accept this "mission" anyways and not be deducted any credits but also have the medicine.

Author: If the player does not have enough credits, the mission will not trigger anyway.
Cross ref:
https://github.com/endless-sky/endless-sky/blob/c9b66ab7fcbdba7fe1cba00f99bfbd83f46a6e14/source/Mission.cpp#L593-L595
https://github.com/endless-sky/endless-sky/blob/c9b66ab7fcbdba7fe1cba00f99bfbd83f46a6e14/source/MissionAction.cpp#L341-L342

		set "james medicine: medicine taken"
		fail

> 	on decline
> 		fail

Reviewer 4: This is basically dead code. There's no mission to "fail" if it was "declined" in the first place. 
```suggestion
```

			label distress
			`	Just before you are overcome by the inexplicable sleepiness, you reach over and hit the big "Distress" button. You can only hope that everything works alright.`
			choice
				`	(Continue.)`
			branch poor
				"net worth" < 30000000
			`	You find yourself awake in a room, which is definitely not your own. It is lit by a luxurious glass chandelier hanging from a lofty groined ceiling. Looking around, there isn't much furniture, apart from the bed, a few scattered chairs, a row of cabinets to the left, and a large display taking nearly the whole wall in front. The floor is made of white and red marble tiles.`
				goto doctor
			label poor
			`	You find yourself awake in a room, which is definitely not your own. It is lit solely by a few windows that reveal a bustling spaceport nearby. There isn't much furniture apart from the bed and a cabinet.`
			label doctor
			`	As you muse and wonder where you are, someone looking like a doctor or perhaps a nurse enters the room.`
			`	"You are lucky to be still living", she says, "You barely survived an encounter with a recently discovered disease tentatively known as X-3625.".`
			choice

> 				`	"Can you tell me more this disease?"`

Reviewer 5: ```suggestion
				`	"Can you tell me more about this disease?"`
```

				`	"Is there anything I should do about it?"`
					goto do
				`	"Yep, just like me. Missing every danger through sheer luck."`
					goto luck

> 			`	"The disease appears to be caused by a certain bacteria of unknown origin. It seems to be quite picky about its surroundings. It thrives best in low gravity environments, gradually spreading over the host's body. When exposed to an environment with higher gravity, such as during landing, the bacteria rapidly swell in size, effectively blocking most of the blood flow and causing a sensation of fatigue and sleepiness."`

Reviewer 2: This would lore the fact that ships have no (or almost no) artificial gravity.
[Discuss here]

Author: If someone has a better reason for the illness to apply only when landing, feel free to comment.

Reviewer 6: How much consistency do we need? Do we care if the player pays attention to the timing? If we don't, we could simply say that it takes a certain amount of time and it was lucky that the player was already landing when it triggered the reaction.

Alternatively, perhaps it is sensitive to changes in magnetic fields. Thus landing on the planet and passing through the magnetic fields trigger a bacterial "bloom."

Author: Magnetic fields are a cool idea indeed. Might switch the explanation to that tomorrow-ish if there aren't other better suggestions.

			choice
				`	"Is there anything I should do about it?"`
					goto do
			label luck
			branch poorluck
				"net worth" < 10000000
			`	"I wouldn't count on luck so much. A risk-averse way of life and a well-written will is worth more than any lucky star out there. But you are a spaceship captain, I guess you wouldn't understand."`
			choice
				`	"Is there anything I should do about this disease?"`
					goto do
			label poorluck
			`	"I wouldn't count on luck so much. A modest way of life and an industrious mentality is worth more than any lucky star out there. But you are a spaceship captain, I guess you wouldn't understand."`
			choice
				`	"Is there anything I should do about this disease?"`
			label do

> 			`	"We already injected an ample amount of the proper medicine. It will not cure you instantly, but it contains certain compounds which inhibit the pathogen's reproduction cycle, causing it to die out soon. Naturally, the life-saving operation and medicine are not free. The cost totals to 100,000 credits."`

Reviewer 2: Endless Sky doesn't believe in Universal Healthcare.

Author: Even if it does, have you paid the respective taxes first? :smiley:
